tm2/pkg/crypto/keys/client/broadcast.go

```diff
@@ -87,6 +88,7 @@ func execBroadcast(cfg *BroadcastCfg, args []string, io commands.IO) error {
 		io.Println("GAS USED:  ", res.DeliverTx.GasUsed)
 		io.Println("HEIGHT:    ", res.Height)
 		io.Println("EVENTS:    ", string(res.DeliverTx.EncodeEvents()))
+		io.Println("TX HASH:   ", hex.EncodeToString(res.Hash))
```

Comment:
We have an "unofficial standard" on how we generate and display tx hashes:

The transaction hash (which is the sha'd amino binary encoding) is **base64 encoded**, and displayed as such throughout all of our tools (ex. Gnoscan, tx-indexer...). We also use this format for the `tx` RPC endpoint

So here, you'd need to base64 it

Reply:
It is worth noting a quirk, we autodecode base64 into bytes on the RPC layer, but the index we save for the transactions is the actual transaction hash in bytes :)

Doesn't change anything in the comment above, just gives additional context

Reply:
Thanks for your info, I will use base64 instead

Reply:
https://github.com/gnolang/gno/pull/2309/commits/c1eb1fe7f6ce25dc3389b905e79b3799312e2c86